shared/featureconfig/config.go

```diff
@@ -268,15 +268,13 @@ func ConfigureValidator(ctx *cli.Context) {
 	complainOnDeprecatedFlags(ctx)
 	cfg := &Flags{}
 	cfg = configureConfig(ctx, cfg)
-	cfg.ProtectProposer = true
-	if ctx.Bool(disableProtectProposerFlag.Name) {
-		log.Warn("Disabled validator proposal slashing protection.")
-		cfg.ProtectProposer = false
-	}
-	cfg.ProtectAttester = true
-	if ctx.Bool(disableProtectAttesterFlag.Name) {
-		log.Warn("Disabled validator attestation slashing protection.")
-		cfg.ProtectAttester = false
+	if ctx.Bool(enableProtectProposerFlag.Name) {
+		log.Warn("Enabled validator proposal slashing protection.")
+		cfg.ProtectProposer = true
+	}
+	if ctx.Bool(enableProtectAttesterFlag.Name) {
+		log.Warn("Enabled validator attestation slashing protection.")
+		cfg.ProtectAttester = true
```

Comment:
These flags just didnt work anymore, protection was always disabled, changing them back to enable flags since they're unused.